test/e2e/diagram.overlays.rendering.test.ts

```diff
@@ -63,6 +63,11 @@ async function addOverlay(bpmnContainerElementHandle: ElementHandle<Element>, bp
   await clickOnButton(position);
 }
 
+async function removeAllOverlays(bpmnContainerElementHandle: ElementHandle<Element>, bpmnElementId: string): Promise<void> {
```

Comment:
⚠️ the `bpmnContainerElementHandle` parameter is unused and should be removed.
I have detected the same issue on the addOverlays. I am working on it in a dedicated PR

Reply:
Good catch :)